estate/models/estate_property.py

```diff
@@ -31,29 +32,66 @@ class Property(models.Model):
         string="Available From",
         default=lambda self: self._default_date_availability(),
         copy=False,
-        help="The property is available starting this date.")
+        help="The property is available starting this date.",
+    )
     expected_price = fields.Float(
         required=True,
-        help="Expected price for the selling of the property (required).")
+        help="Expected price for the selling of the property (required).",
+    )
     selling_price = fields.Float(
         readonly=True,
         copy=False,
-        help="Price at which the property is sold.")
+        help="Price at which the property is sold.",
+    )
     bedrooms = fields.Integer(
         default=2,
-        help="Integer number of bedrooms of the property.")
+        help="Integer number of bedrooms of the property.",
+    )
     living_area = fields.Integer(
         string="Living Area (sqm)",
-        help="Integer number of sqm of living area of the property.")
+        help="Integer number of sqm of living area of the property.",
+    )
     facades = fields.Integer(help="Integer number of facades of the property.")
     garage = fields.Boolean(help="Marked if the property has garage.")
     garden = fields.Boolean(help="Marked if the property has garden.")
     garden_area = fields.Integer(
         string="Garden Area (sqm)",
-        help="Integer area in sqm of the garden area.")
+        help="Integer area in sqm of the garden area.",
+    )
     garden_orientation = fields.Selection(
-        selection=[("north", "North"), ("south", "South"), ("east", "East"), ("west", "West")],
-        help="Orientation of the garden."
+        selection=[
+            ("north", "North"),
+            ("south", "South"),
+            ("east", "East"),
+            ("west", "West"),
+        ],
+        help="Orientation of the garden.",
+    )
+    property_type_id = fields.Many2one(
+        "estate.property.type",
+        copy=False,
```

Comment:
It is necessary?

Reply:
I got confused and added to the wrong field. 😅 😆